sdks/java/core/src/main/java/org/apache/beam/sdk/testing/TestPipeline.java

```diff
@@ -238,7 +245,7 @@ protected void afterUserCodeFinished() {
     }
   }
 
-  static final String PROPERTY_BEAM_TEST_PIPELINE_OPTIONS = "beamTestPipelineOptions";
+  public static final String PROPERTY_BEAM_TEST_PIPELINE_OPTIONS = "beamTestPipelineOptions";
```

Comment:
It makes sense that actually both of these properties should be public, since they reach all the way to the world outside Java.

Javadoc?